modules/administration/pages/content-lifecycle-examples.adoc

```diff
@@ -0,0 +1,260 @@
+For example, [systemitem]``Include kernel security fix``.
+
+. In the [guimenu]``Filter Type`` field, select [guimenu]``Patch (Advisory Name)``.
+
+. In the [guimenu]``Matcher`` field, select either [guimenu]``equals`` or [guimenu]``matches``.
+
+. In the [guimenu]``Advisory Name`` field, type [guimenu]``SUSE-15-2019-2071``, and check [guimenu]``Allow``.
+
+. Click btn:[Save] to store the filter.
+
+. Navigate to menu:Content Lifecycle[Projects] and select your project from the list.
+
+. Click btn:[Attach/Detach Filters], and select [guimenu]``Include kernel security patch``.
+
+. Click btn:[Save].
+
+. Click btn:[Build] to rebuild the environment.
+
+
+
+[#update-a-patch-cycle]
+== Update an Existing Monthly Patch Cycle
+
+When a monthly patch cycle is complete, you can update the patch cycle for the next month.
+
+.Procedure: Updating a Monthly Patch Cycle
+. In the [guimenu]``by date`` field, change the date of the filter to the next month.
+Alternatively, create a new filter and change the assignment to the project.
+
+. Check if the exclude filter for ``SUSE-15-2019-1807`` can be detached from the project.
+There may be a new patch available to fix this issue.
+
+. Detach the ``include`` filter. 
+The filter is included by default.
+
+. Complete updating your filters.
+. Rebuild the project to create a new environment with patches for 1 September.
+
+
+
+[#enhance-project-with-livepatching]
+== Enhance a Project with Livepatching
+
+This section covers setting up filters to create environments for live patching.
+
+[IMPORTANT]
+====
+
+When you are preparing to use live patching, there are some important considerations
+
+* Only ever use one kernel version on your systems. 
+The live patching packages are installed with a specific kernel.
+
+* Live patching updates are shipped as one patch.
+
+* Each kernel patch that begins a new series of live patching kernels will display the ``required reboot`` flag.
+These kernel patches come with live patching tools. 
+When you have installed them, you will need to reboot the system at least once before the next year.
+
+* Only install live patch updates that match the installed kernel version.
+
+* Live patches are provided as stand-alone patches. 
+You must exclude all regular kernel patches with higher kernel version than the currently installed one.
+====
+
+
+
+[#exclude-higher-kernel-version]
+=== Exclude Packages with a Higher Kernel Version
+
+In this example you update your systems with the ``SUSE-15-2019-1244`` patch.
+This patch contains ``kernel-default-4.12.14-150.17.1-x86_64``.
+
+You need to exclude all patches which contain a higher version of ``kernel-default``.
+
+Procedure: Excluding Packages with a Higher Kernel Version
+
+. In the {productname} {webui}, navigate to menu:Content Lifecycle[Filters], and click btn:[Create Filter].
+
+. In the [guimenu]``Filter Name`` field, type a name for your filter.
+For example, [systemitem]``Exclude kernel greater than 4.12.14-150.17.1``
```

Comment:
```suggestion
For example, [systemitem]``Exclude kernel greater than 4.12.14-150.17.1``.
```

Reply:
@keichwa we can always rely on your to find the missing periods! Nice work! :heart: